nodemon.json

```diff
@@ -0,0 +1,9 @@
+{
+ "ignore":[
+    ".git",
+    "node_modules/**/node_modules",
```

Comment:
Can you help me understand the need for this glob? I believe nodemon automatically  excludes node_modules.
"Note that by default, nodemon will ignore the .git, node_modules, bower_components, .nyc_output, coverage and .sass-cache directories and add your ignored patterns to the list"

Reply:
Oh, I think you're right. To be honest I took that from the nodemon example json that you posted [here](https://github.com/remy/nodemon/blob/master/doc/sample-nodemon.md). I assumed that once I specified my ignore list that it would override the default. I didn't realize that it would be appended and not replaced. That's probably safe to remove. Thanks for noticing that